src/components/ResultCard/ResultCard.tsx

```diff
@@ -51,18 +51,35 @@ export default function ResultCard(props: ResultCardProps) {
   );
 
   const resultCardContentWithLink = () => (
-    <a className='cio-result-card-anchor' href={result.data?.url}>
-      {resultCardContent()}
-    </a>
+    <>
+      <a
+        onClick={() => clickHandler()}
+        onKeyDown={(e) => keyDownHandler(e)}
+        className='cio-result-card-image'
+        href={result.data?.url}>
+        <img src={result.data?.image_url} alt='product' />
+      </a>
+      <a
+        onClick={() => clickHandler()}
+        onKeyDown={(e) => keyDownHandler(e)}
+        className='cio-result-card-text'
+        href={result.data?.url}>
+        <p className='cio-result-card-title'>{result.value}</p>
+        <div className='cio-result-card-prices'>
+          {salePrice && <span className='cio-result-card-sale-price'>${salePrice}</span>}
+          {regularPrice && (
+            <span className={`cio-result-card-regular-price${salePrice ? '--strike-through' : ''}`}>
+              ${regularPrice}
+            </span>
+          )}
+        </div>
+      </a>
+      <ResultCtaButton item={result} price={salePrice || regularPrice} />
+    </>
   );
 
   return (
-    <div
-      onClick={() => clickHandler()}
-      onKeyDown={(e) => keyDownHandler(e)}
-      className='cio-result-card'
-      role='button'
-      tabIndex={0}>
+    <div className='cio-result-card'>
```

Comment:
Why did we remove the events from here? 

--------------

I think the previous behavior was: 
If there is a custom click handler -> Use that instead of redirecting to URL
If there is no custom click handler -> Redirect to URL

--------------

Current behavior:
If there is a custom click handler -> No click action (because we use `resultCardContent` and div with no listeners)
If there is no custom click handler -> Redirect to URL

--------------

If we update the next line to `customClickItemCallback` instead of `! customClickItemCallback`:
If there is a custom click handler -> Call the custom click handler + redirect to URL (we probably don't want to redirect here)
If there is no custom click handler -> No click action (we want the redirect here)